tests/helix/send-to-helix-basictests.targets

```diff
@@ -3,9 +3,15 @@
   <PropertyGroup>
     <WorkItemArchiveWildCard>$(TestArchiveTestsDir)**/*.zip</WorkItemArchiveWildCard>
     <BuildHelixWorkItemsDependsOn>$(BuildHelixWorkItemsDependsOn);BuildHelixWorkItemsForDefaultTests</BuildHelixWorkItemsDependsOn>
+    <NeedsDcpPathOverride>true</NeedsDcpPathOverride>
   </PropertyGroup>
 
   <Target Name="BuildHelixWorkItemsForDefaultTests">
+    <ItemGroup>
+      <!-- needed for Aspire.Hosting.Container.Tests -->
+      <HelixPreCommand Include="$(_EnvVarSetKeyword) DOCKER_BUILDKIT=1" />
```

Comment:
Is it possible to put this in the Hosting.Container.Tests so it doesn't need to be set everywhere?

Reply:
Would that be useful? Right now it would have to be by conditionally adding it on the work item itself.

Reply:
My thinking is that we wouldn't need a comment like `<!-- needed for Aspire.Hosting.Container.Tests -->` if it was in the Aspire.Hosting.Container.Tests project.

Reply:
Agreed. In the follow up I'm generating run scripts which would allow individual test projects to control their own setups directly.